spring-webmvc/src/main/java/org/springframework/web/servlet/handler/AbstractHandlerExceptionResolver.java

```diff
@@ -255,7 +255,7 @@ protected void logException(Exception ex, HttpServletRequest request) {
 	 * @return the log message to use
 	 */
 	protected String buildLogMessage(Exception ex, HttpServletRequest request) {
-		return "Resolved [" + LogFormatUtils.formatValue(ex, -1, true) + "]";
+		return "Resolved [" + LogFormatUtils.formatValue(ex, -1, true) + "] :" + ex.getMessage();
```

Comment:
This should be in the formatValue call:
```suggestion
		return "Resolved [" + LogFormatUtils.formatValue(ex + "] :" + ex.getMessage(), -1, true);
```